tests/product/tests/data/k8s_object/eks_k8s_object_test_cases.py

```diff
@@ -160,3 +161,67 @@
     '4.2.9 PSP spec.securityContext.capabilities.drop==["ALL"] eval passed': cis_eks_4_2_9_pass,
     '4.2.9 PSP spec.securityContext.capabilities.add==["NET_ADMIN", "SYS_TIME"] eval failed': cis_eks_4_2_9_fail,
 }
+
+k8s_object_config_1 = {
```

Comment:
same comment for the zip + unpack format

Reply:
A new [task](https://github.com/elastic/security-team/issues/5758) was opened